packages/forma-36-react-components/tools/plop-templates/components/stories.tsx.hbs

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text } from '@storybook/addon-knobs';
 
-import {{pascalCase name}} from './{{pascalCase name}}';
+import {{pascalCase name}}, { {{pascalCase name}}Props } from './{{pascalCase name}}';
+import notes from './README.mdx';
 
-storiesOf('Components/{{pascalCase name}}', module)
-.addParameters({
-propTypes: {{pascalCase name}}['__docgenInfo'],
-component: {{pascalCase name}},
-})
-.add('default', () => (
-<{{pascalCase name}} className={text('className', '' )}>
-  {{{pascalCase name}}}
-</{{pascalCase name}}>
-));
\ No newline at end of file
+
+export default {
+  title: 'Components/{{pascalCase name}}',
+  component: {{pascalCase name}},
+  parameters: {
+    propTypes: [{{pascalCase name}}['__docgenInfo']],
+    notes,
+  },
+  argTypes: {
+    testId: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: {{pascalCase name}}Props) => (
+  <{{pascalCase name}}
+      testId={args.testId}
+    />
+);
+
+basic.args = { testId: 'testId' };
```

Comment:
I think we don't need this because testId has a default value already

Reply:
ok. I thought it would be helpful for someone who is new to this format. No worries, you can always have a look how it's done in other components. I will remove it.